src/probnum/linalg/solvers/policies/_conjugate_gradient.py

```diff
@@ -0,0 +1,31 @@
+"""Policy returning :math:`A`-conjugate actions."""
+
+import numpy as np
+
+import probnum  # pylint: disable="unused-import"
+
+from . import _linear_solver_policy
+
+
+class ConjugateGradientPolicy(_linear_solver_policy.LinearSolverPolicy):
+    r"""Policy returning :math:`A`-conjugate actions.
+
+    Selects the negative gradient / residual as an initial action :math:`s_0 = A x_0 - b` and then successively generates :math:`A`-conjugate actions, i.e. the actions satisfy :math:`s_i^\top A s_j = 0` iff :math:`i \neq j`.
+    """
+
+    def __call__(
+        self, solver_state: "probnum.linalg.solvers.LinearSolverState"
+    ) -> np.ndarray:
+
+        action = -solver_state.residual.copy()
+
+        if solver_state.step > 0:
+            # A-conjugacy correction (in exact arithmetic)
+            beta = (
+                np.linalg.norm(solver_state.residual)
```

Comment:
This can be cached and reused in the stopping criteria

Reply:
As discussed the cache in `LinearSolverState` will be handled in a future PR.